Regression/WarpX-tests.ini

```
@@ -2216,6 +2216,22 @@ compileTest = 0
 doVis = 0
 analysisRoutine = Examples/Modules/nuclear_fusion/analysis_deuterium_tritium_fusion.py
 
+[Deuterium_Tritium_Fusion_RZ]
+buildDir = .
+inputFile = Examples/Modules/nuclear_fusion/inputs_deuterium_tritium_rz
+runtime_params = warpx.do_dynamic_scheduling=0 warpx.serialize_initial_conditions=1
+dim = 2
+addToCompileString =
+cmakeSetupOpts = -DWarpX_DIMS=RZ
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 1
+numthreads = 2
```

I think this needs to be maximum 1 if `numprocs = 2`:
```suggestion
numthreads = 1
```